### ISSUE TITLE
The extension view hover cannot be accessed by the mouse

### ISSUE THREAD

**Issue by mrlitong** [microsoft/vscode#163041]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 版本: 1.72.0
- OS Version: MacOS Monterey Version 12.6 (21G115)

Steps to Reproduce:
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/13718066/194696843-a608abfe-3af1-4b40-afb6-ca69b96dfeb2.png">

1. Install a deprecated extension, here I chose [Bracket Pair Colorizer 2](https://marketplace.visualstudio.com/items?itemName=CoenraadS.bracket-pair-colorizer-2)
2. Move your cursor to the warning icon, and there will be a pop-up windows, like the screenshot.
3. After step 2, you wanna click the 「设置」Hyperlink。（meanings 「setting」in English），you gotta try a lot of times before you can get access to it, because the pop-up windows will disappear when your cursor move.

**Comment by Tyriar** [comment]
I think we have an issue tracking this but I couldn't find it

**Comment by Tyriar** [comment]
For any hover you can hold alt to lock it in place.https://code.visualstudio.com/docs/editor/accessibility#_hover-accessibility

I'll pass this back to @sandy081 as there's now an `appearance.showHoverHint` property that can be used to display a hint on how to lock the hover in place that can be used when it's most useful:

![image](https://github.com/microsoft/vscode/assets/2193314/fc728994-88c9-442d-83e8-7e414f691efa)

**Comment by sandy081** [comment]
Closing this given that `alt` key support to lock the hover. If more users find the similar issue, then we can show the hint.
